### PR TITLE
Add an option for WP_DEBUG

### PR DESCRIPTION
**Pull request by karellm**
A quick improvement that let you set `WP_CONFIG`. I needed that while developing a theme

### CODE REVIEW

**Comment by karellm** [comment]
@dsifford Can you review this? Thanks

**Comment by dsifford** [comment]
@karellm Looks alright. 

What about doing it like this instead? https://github.com/wp-cli/wp-cli/blob/4952290c9b6d14d938ac8f9fbf3f9439df83573b/php/commands/core.php#L324

Your call. I'll be able to test tomorrow.

**Comment by karellm** [comment]
What's the benefit? I was thinking you might want to switch it on and off. And `wp core config` would work only once (though we can remove and regenerate it).

**Comment by dsifford** [comment]
No benefit. Didn't know it was something u wanted switched on and off.

**Comment by karellm** [comment]
@dsifford Ok let me know if we are good to merge!

File: run.sh [hunk 1]
```diff
@@ -76,6 +79,9 @@ else
   printf "Already exists!\n"
 fi
 
+# Enable or Disable debug mode
```
Comment (dsifford) [inline run.sh]: What's the purpose of this?

File: run.sh [hunk 2]
```diff
@@ -28,6 +29,8 @@ core config:
   dbpass: $DB_PASS
   dbname: $DB_NAME
   dbhost: db:3306
+  extra-php: |
+    define('WP_DEBUG', ${WP_DEBUG,,});
```
Comment (dsifford) [inline run.sh]: Nothing wrong here, just a question for ya...

What does the `,,` do in `${WP_DEBUG,,}`? I've never seen that before. Can you explain? :smile:

Comment (karellm) [inline run.sh]: It returns the lowercase version of `WP_DEBUG`. For some reason docker-compose would capitablize the environment variable.

File: run.sh (hunk 1 above)
Comment (karellm) [inline run.sh]: `wp-cli.yml` is generated only once. We need this to subsequently change it.

Comment (dsifford) [inline run.sh]: Not sure if I'm understanding.. Why is this needed if you're explicitly setting WP_debug with `wp-cli.yml`? If I'm reading this correctly, won't this just set the same value twice? Am I missing something?

If the goal is to generate a new `wp-config` file after build success (using a different option than was set initially), why not just write an npm run command to delete the current config file and then regenerate a new one using `wp core config [...]`?

Comment (karellm) [inline run.sh]: On first run yes. But if you stop the container and start it again with the ENV variable, the `wp-cli.yml` won't get generated a second time. But I like your idea of a script I think

Comment (dsifford) [inline run.sh]: Ah, ok I understand now. Makes sense I suppose.

**Comment by karellm** [comment]
Merging, we can work on replacing with a script once we figure out what we want on that level.

**Pull request merged**